packages/near-membrane-base/src/instrumentation.ts

```diff
@@ -0,0 +1,12 @@
+export type InstrumentationDataType = object | string | number | boolean;
+
+export interface InstrumentationHooks {
+    startActivity(activityName: string, data?: InstrumentationDataType): Activity;
+    log(data: InstrumentationDataType): void;
+    error(data: InstrumentationDataType): void;
+}
+
+export interface Activity {
+    stop(data?: InstrumentationDataType): void;
+    error(data?: InstrumentationDataType): void;
+}
```

Comment:
It would nice if we could align our instrumentation APIs as much as possible; right now Locker's interface looks like: 

```ts
export interface LockerActivity {
    start: () => LockerActivity;
    stop: () => LockerActivity;
}
// Locker specific instrumentation interface
// Defines internal beacons used by Locker only
export interface LockerInstrumentation {
    activityBeacon: (sandboxKey: string, activityNameOrDescription: string) => LockerActivity;
    errorBeacon: (sandboxKey: string, e: Error) => void;
    logBeacon: (sandboxKey: string, message: string) => void;
}
```

...But there are things here that don't make sense for near-membrane.

Reply:
Yes. The plan is to remove those interface above and use the new ones in near-membrane after this PR is merged.